fmriprep/workflows/bold/base.py

```diff
@@ -516,20 +506,34 @@ def init_func_preproc_wf(bold_file):
                 ('outputnode.bold', 'inputnode.bold')]),
             (bold_split, bold_t1_trans_wf, [
                 ('out_files', 'inputnode.bold_split')]),
+            (bold_hmc_wf, bold_t1_trans_wf, [
+                ('outputnode.xforms', 'inputnode.hmc_xforms')]),
+            (bold_sdc_wf, bold_t1_trans_wf, [
+                ('outputnode.out_warp', 'inputnode.fieldwarp')])
         ])
-    else:  # for meepi, create and use optimal combination
+    else:  # for meepi, use optimal combination
         workflow.connect([
             # update name source for optimal combination
             (inputnode, func_derivatives_wf, [
                 (('bold_file', combine_meepi_source), 'inputnode.source_file')]),
             (bold_bold_trans_wf, skullstrip_bold_wf, [
                 ('outputnode.bold', 'inputnode.in_file')]),
+            (skullstrip_bold_wf, join_echos, [
+                ('outputnode.skull_stripped_file', 'bold_files')]),
+            (join_echos, bold_t2s_wf, [
+                ('bold_files', 'inputnode.bold_file')]),
             (bold_t2s_wf, bold_confounds_wf, [
                 ('outputnode.bold', 'inputnode.bold')]),
-            (bold_t2s_wf, bold_t1_trans_wf, [
-                ('outputnode.bold', 'inputnode.bold_split')]),
+            (bold_t2s_wf, split_opt_comb, [
+                ('outputnode.bold', 'in_file')]),
+            (split_opt_comb, bold_t1_trans_wf, [
+                ('out_files', 'inputnode.bold_split')]),
         ])
 
+        # Already applied in bold_bold_trans_wf, which inputs to bold_t2s_wf
+        bold_t1_trans_wf.inputs.inputnode.fieldwarp = 'identity'
+        bold_t1_trans_wf.inputs.inputnode.hmc_xforms = 'identity'
```

Comment:
@tsalo HMC and SDC are not being reapplied in bold_t1_trans_wf in the multi-echo case.